components/admin_console/database_settings.jsx

```diff
@@ -240,6 +242,25 @@ export default class DatabaseSettings extends AdminSettings {
                     onChange={this.handleChange}
                     setByEnv={this.isSetByEnv('SqlSettings.ConnMaxLifetimeMilliseconds')}
                 />
+                <TextSetting
+                    id='minHashtagLength'
+                    label={
+                        <FormattedMessage
+                            id='admin.service.minHashtagLengthTitle'
+                            defaultMessage='Minimum Hashtag Length:'
+                        />
+                    }
+                    placeholder={Utils.localizeMessage('admin.service.minHashtagLengthExample', 'E.g.: "3"')}
+                    helpText={
+                        <FormattedMessage
+                            id='admin.service.minHashtagLengthDescription'
+                            defaultMessage='Minimum number of characters in a hashtag. MySQL data bases must be configured to support searching strings shorter than three characters, [see documentation](!https://dev.mysql.com/doc/refman/8.0/en/fulltext-fine-tuning.html).'
```

Comment:
I think I'm missing something here since there is no hyperlink 😅

Reply:
You'll want to use a `FormattedMarkdownMessage` here instead of a `FormattedMessage` for that to work. `FormattedMarkdownMessage` comes from `components/formatted_markdown_message`